nixos/modules/services/security/vaultwarden/default.nix

```diff
@@ -57,6 +57,14 @@ in {
       '';
     };
 
+    stateDir = mkOption {
```

Comment:
How is this directory created? See https://github.com/NixOS/nixpkgs/blob/fd8a7fd07da0f3fc0e27575891f45c2f88e5dd44/nixos/modules/services/misc/gitolite.nix#L165-L177 as an example.